scripts/Run-BisectTest.ps1

```diff
@@ -9,15 +9,23 @@ param (
     [Parameter(Mandatory = $false,
         ParameterSetName = 'BisectStart')]
     [string[]]$Skip,
-    [switch]$Force    
+    [switch]$Force,
+    [switch]$FirstParent
```

Comment:
It would be great if you could add script docs explaining each parameter, see https://learn.microsoft.com/powershell/scripting/developer/help/comment-based-help-keywords

Reply:
Sure.  Will do.